services/spiget/spiget-download-size.tester.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+const Joi = require('joi')
+const { isFileSize } = require('../test-validators')
+
+const t = (module.exports = require('../create-service-tester')())
+
+t.create('EssentialsX (id 9089)')
+  .get('/9089.json')
+  .expectJSONTypes(Joi.object().keys({ name: 'size', value: isFileSize }))
+
+t.create('Invalid Resource (id 1)')
+  .get('/1.json')
+  .expectJSONTypes(Joi.object().keys({ name: 'size', value: 'not found' }))
```

Comment:
When we know **exactly** what the response will be, we can and should use the `.expectJSON` function:

```js
.expectJSON({
  name: 'size',
  value: 'not found',
})
```